src/pages/Cart/Cart.js

```diff
@@ -1,22 +1,88 @@
-import { React, useState, useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import CartBox from './component/CartBox';
 import './Cart.scss';
+import { useNavigate } from 'react-router-dom';
 
 const Cart = () => {
+  const [productList, setProductList] = useState([]);
+  const [cartList, setCartList] = useState([]);
+  const [isAlertOpen, setIsAlertOpen] = useState(false);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    fetch('/data/cartData.json')
+      .then(res => res.json())
+      .then(data => {
+        setProductList(data);
+      });
+  }, []);
+
+  // useEffect(() => {
+  //   fetch('http://10.58.52.117:3000/carts', {
+  //     method: 'GET',
+  //     headers: { Authorization: localStorage.getItem('token') },
+  //   })
+  //     .then(res => res.json())
+  //     .then(data => {
+  //       setCartList(data.carts);
+  //     });
+  // }, []);
```

Comment:
```suggestion
  // TODO : 차후 실제 통신을 통해 사용할 코드
  // useEffect(() => {
  //   fetch('http://10.58.52.117:3000/carts', {
  //     method: 'GET',
  //     headers: { Authorization: localStorage.getItem('token') },
  //   })
  //     .then(res => res.json())
  //     .then(data => {
  //       setCartList(data.carts);
  //     });
  // }, []);
```

주석 처리 해서 남겨야 할 코드가 있다면 꼭 해당 주석이 어떤 의미인지 남겨주세요!